src/main/java/jenkins/plugins/nodejs/tools/NodeJSInstallation.java

```diff
@@ -61,7 +69,9 @@ public NodeJSInstallation forNode(Node node, TaskListener log) throws IOExceptio
     }
 
     public String getExecutable(Launcher launcher) throws InterruptedException, IOException {
-        return launcher.getChannel().call(new Callable<String, IOException>() {
+        return launcher.getChannel().call(new MasterToSlaveCallable<String, IOException>() {
+            private static final long serialVersionUID = 1L;
```

Comment:
Should be made into a `static` nested class.

(That problem existed before your PR, of course, but if you are going to bother adding a `serialVersionUID` you should do it right.)